app/views/providers/allocations/pick_a_provider.html.erb

```diff
@@ -0,0 +1,65 @@
+<%= content_for :page_title, "Pick a provider"%>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(initial_request_provider_recruitment_cycle_allocations_path(@provider.provider_code, @recruitment_cycle.year)) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl" data-qa="heading">Pick a provider</h1>
+
+    <p class="govuk-body">You searched for ‘<%= params[:training_provider_query] %>’</p>
+    <p class="govuk-body">We found these providers which matched your search:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <% training_providers.each do |training_provider| %>
+        <li data-qa="provider_suggestion">
+          <%= govuk_link_to "#{training_provider.provider_name} (#{training_provider.provider_code})",
+            initial_request_provider_recruitment_cycle_allocations_path(training_provider_code: training_provider.provider_code), { class: "govuk-link" }
+          %>
+        </li>
+      <% end %>
+    </ul>
+
+    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
+    <div data-qa="search">
+      <details class="govuk-details" role="group" data-qa="search__expand">
+        <summary class="govuk-details__summary" role="button">
+          <span class="govuk-details__summary-text">Try another provider</span>
+        </summary>
+
+        <div class="govuk-details__text">
+          <%= form_with url: initial_request_provider_recruitment_cycle_allocations_path(
+                          @provider.provider_code,
+                          @provider.recruitment_cycle_year,
+                          training_provider_code: params[:training_provider_code]
+                        ),
+                        skip_enforcing_utf8: true,
+                        method: :get,
+                        builder: GOVUKDesignSystemFormBuilder::FormBuilder do |form| %>
+            <%= form.hidden_field :training_provider_code, value: "-1" %>
+
+            <fieldset class="govuk-fieldset">
```

Comment:
I don't think this needs to be a fieldset

Reply:
The fieldset is now gone